Reference/google/protobuf/any_test.pb.swift

```diff
@@ -123,35 +108,42 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Proto3Message, SwiftProtobuf._Mes
   init() {}
 
   mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
-    try _uniqueStorage().decodeMessage(decoder: &decoder)
+    try withExtendedLifetime(_storage) { (_storage: _StorageClass) in
+      while let fieldNumber = try decoder.nextFieldNumber() {
+        try decodeField(decoder: &decoder, fieldNumber: fieldNumber)
+      }
+    }
   }
 
   mutating func _protoc_generated_decodeField<D: SwiftProtobuf.Decoder>(decoder: inout D, fieldNumber: Int) throws {
-    try _uniqueStorage().decodeField(decoder: &decoder, fieldNumber: fieldNumber)
+    switch fieldNumber {
```

Comment:
Good point... that might have gotten left on the floor and wasn't caught because the CoW tests we have didn't have to deal with merging at the time.

I'll write some tests and fix it up. It's going to suck to have to call `_uniqueStorage()` on every `decodeField` call when we know that we really only need to call it around `decodeMessage` in most cases, but groups calling into `decodeField` directly make that not possible. But I guess again, if groups are rare, the performance penalty won't hit most people.